section-3/exercises/vsnitsar-section3.js

```diff
@@ -0,0 +1,182 @@
+2. Create an array with different data types: a string, a number, and a boolean. Print its length.
+3. Create an array animals with at least 3 elements. Print the second element.
+4. Change the last element of animals to "Lion" and print the updated array.
+*/
+
+console.log("Exercise 1.1.")
+const numbers = [];
+numbers[0] = 10;
+numbers[1] = 20;
+numbers[numbers.length] = 30;
+
+console.log("Exercise 1.2.")
+const mixedArray = ['sea', 33, true]
+console.log(mixedArray.length);
+
+console.log("Exercise 1.3.")
+const animals = ['Lion', 'Elephant', 'Dog', 'Cat', 'Puma']
+console.log(animals[1]);
+
+console.log("Exercise 1.4.")
+animals[animals.length - 1] = 'Lion'
+console.log(animals);
+
+/*
+Exercise 2:
+1. Create an object named car with properties: brand, model, and year. Print the whole object
+2. Add properties *color* and *engine* to the car object by different ways. Set values to that
+properties what ever you want.
+3. Change the value of year to 2025 and print the updated object.
+4. Delete brand property from object and print the updated object
+*/
+
+console.log("Exercise 2.1.")
+const car = {
+    brand: 'Peugeot',
+    model: '2008',
+    year: '2023',
+}
+console.log(car);
+
+console.log("Exercise 2.2.")
+console.log("Method 1:");
+car.color = 'Blue';
+console.log(car)
+console.log("Method 2:");
+Object.assign(car, {engine: '1.2 Turbo'});
+console.log(car)
+console.log("Method 3:");
+const anotherCar = {
+    wheels: '17'
+}
+latestCar = {...car, ...anotherCar}
+console.log(latestCar);
+
+console.log("Exercise 2.3.")
+car.year = 2025;
+console.log(car);
+
+console.log("Exercise 2.4.")
+delete car.brand;
+console.log(car);
+
+/*
+Exercise 3:
+Create an array cars with "Toyota", "Ford", "BMW".
+1. Add "Tesla" to the end of array.
+2. Remove "Toyota" car from cars array and print the array.
+3. Check if array includes "Ford" in cars array
+4. Add Mercedes as a 1st element of cars array
+*/
+
+console.log("Exercise 3.1.")
+const cars = ['Toyota', 'Ford', 'BMW'];
+cars.push('Tesla');
+
+console.log("Exercise 3.2.")
+cars.splice(0, 1);
+console.log(cars);
+
+console.log("Exercise 3.3.")
+console.log("Cars array includes 'Ford': ", cars.includes('Ford'));
+console.log(`Cars array includes 'Ford': ${cars.includes('Ford')}`);
+
+console.log("Exercise 3.4.")
+cars.unshift('Mercedes');
+
+/*
+Exercise 4:
+What will show console log in the code bellow ? Think without code execution
+    const cars = ["Toyota", "Ford", "BMW", 'Tesla'];
+    const carsTwo = cars;
+    carsTwo.push('Mercedes');
+    console.log(cars.length);
+*/
+
+//Answer: both will be changed because of reference type of array, and both will have length 5
+
+/*
+Exercise 5:
+Create an object calculator with properties num1 and num2, add method add() that returns their sum.
+*/
+
+const calculator = {
+    add: function (num1, num2) {
+        this.num1 = num1;
+        this.num2 = num2;
+        return this.num1 + this.num2;
```

Comment:
In the exercise you should create calculator object with properties numOne & numTwo, each property should have a number value. Create a method **add** which will return sum of properties.